packages/nuxt/src/module.ts

```diff
@@ -5,7 +5,7 @@ import {
   defineNuxtModule,
   addPlugin,
   isNuxt2,
-  addAutoImport,
+  addImports,
   createResolver,
   resolveModule,
```

Comment:
Note: This utility is deprecated from `@nuxt/kit` as is CJS-based resolution. But should be fine for pinia module.

Reply:
Should I use `const runtimeDir = fileURLToPath(new URL('./runtime', import.meta.url))` + `resolve()`?